docs/UserGuide.md

```diff
@@ -320,6 +320,9 @@ Have a growing list of internships and finding it challenging to prioritize? Or
 ℹ️ We do not allow you to stack sorts. A subsequent `sort` command will replace the prior one. Otherwise, the initial sort remains effective.
 </div>
 
+<div markdown="block" class="alert alert-info">
+ℹ️ We do not allow you to sort by requirements.
+</div>
```

Comment:
This is a good start but it's not enough. Can you explain in 1 or 2 sentences why we prohibit this (to guard against attacks in actual PE). Don't be too wordy too though if not we get penalised in 2101